src/lua/events.c

```diff
@@ -633,9 +630,6 @@ static int destroy_shortcut_event(lua_State *L)
   // remove the accelerator from the lua shortcuts
   dt_accel_deregister_lua(tmp);
 
-  // free temporary buffer
-  free(tmp);
```

Comment:
In contrast to the situation above, the `free` _is_ needed here (as far as I can see) as there is closure that takes care for freeing `tmp`.

Reply:
I'll try and trace where it's getting freed.

Reply:
@rabauke after lots of testing I agree, this should be freed.  Do you want to do the PR or do you want me to?

Reply:
@wpferguson I can create a new pull request. No problem.